packages/tools/kadena-cli/src/dapp/commands/dappCreate.ts

```diff
@@ -0,0 +1,38 @@
+import chalk from 'chalk';
+import { spawnSync } from 'child_process';
+import type { Command } from 'commander';
+import debug from 'debug';
+import { existsSync } from 'fs';
+import { join } from 'path';
+
+import { createCommand } from '../../utils/createCommand.js';
+import { globalOptions } from '../../utils/globalOptions.js';
+
+export const createDappCommand: (program: Command, version: string) => void =
+  createCommand(
+    'create',
+    'create a new dapp project',
+    [globalOptions.dappTemplate()],
+    async (config, args) => {
+      const projectDir = join(process.cwd(), args[0]);
+      const { dappTemplate } = config;
+      const folderExists = existsSync(projectDir);
```

Comment:
please use `services.filesystem.directoryExists` instead, we have this for testing purposes